5/alpine/Dockerfile

```diff
@@ -13,9 +13,9 @@ ENV GPG_KEY 46095ACC8548582C1A2699A9D27D666CD88E42B4
 WORKDIR /usr/share/elasticsearch
 ENV PATH /usr/share/elasticsearch/bin:$PATH
 
-ENV ELASTICSEARCH_VERSION 5.2.2
-ENV ELASTICSEARCH_TARBALL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.2.2.tar.gz" \
-	ELASTICSEARCH_TARBALL_ASC="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.2.2.tar.gz.asc" \
+ENV ELASTICSEARCH_VERSION 5.3.0
+ENV ELASTICSEARCH_TARBALL="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.3.0.tar.gz" \
+	ELASTICSEARCH_TARBALL_ASC="https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.3.0.tar.gz.asc" \
 	ELASTICSEARCH_TARBALL_SHA1="2b19e0e334db8880e352c392a52d464be3d8bc0b"
```

Comment:
This line would also need to be updated (see the failure in the travis build).  But all of this is actually covered in by just running `update.sh`. We have this on a periodic jenkins job. but elasticsearch 5.3 was failing in alpine when we run `elasicsearch --version` with the output `Killed`, so it was likely running out of memory. :cry:

I'll look to fixing our jenkins job so that this will be automatic in the future, but in the meantime would you update your PR with the new SHA?